src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java

```diff
@@ -3086,4 +3219,31 @@ public String toString() {
                + ", \n ocpModel = " + ocpModel + "\n}\n";
     }
 
+    public ConcurrentHashMap<String, String> getTableGroupInverted() {
+        return TableGroupInverted;
+    }
+
+    public ConcurrentHashMap<String, String> getTableGroupCache() {
+        return TableGroupCache;
+    }
+
+    /**
+     * get table route fail than clear table group message
+     * @param phyTableName
+     * @param tableGroupName
+     */
+    public void clearTableGroupCache(String phyTableName, String tableGroupName) {
```

Comment:
tableGroupName is enough
BTW, 'clean'/'delete'/'erase' are better than 'clear'
`eraseTableGroupFromCache(String tableGroupName);`